From ad30a0061be5b77c6c3be39023df1898eae9a307 Mon Sep 17 00:00:00 2001
From: Herrie <Github.com@herrie.org>
Date: Wed, 3 Jan 2024 14:17:04 +0100
Subject: [PATCH] luna-sysservice: TimePrefsHandler.cpp: Fix typo

mmcInfo should be mccInfo as per legacy webOS.

Typo was introduced in OWO and still present in OSE.

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
---
Upstream-Status: Pending

 Src/TimePrefsHandler.cpp | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/Src/TimePrefsHandler.cpp b/Src/TimePrefsHandler.cpp
index ef945a0..95ee2ef 100755
--- a/Src/TimePrefsHandler.cpp
+++ b/Src/TimePrefsHandler.cpp
@@ -871,9 +871,9 @@ JValue TimePrefsHandler::timeZoneListAsJson(const std::string& countryCode, cons
 			break;
 		}
 
-		JValue mmcInfoObj = TimePrefsHandler::s_timeZonesJson["mmcInfo"];
-		if (!mmcInfoObj.isObject()) {
-			qWarning() << "Failed to parse mmcInfo details";
+		JValue mccInfoObj = TimePrefsHandler::s_timeZonesJson["mccInfo"];
+		if (!mccInfoObj.isObject()) {
+			qWarning() << "Failed to parse mccInfo details";
 			break;
 		}
 
@@ -913,7 +913,7 @@ JValue TimePrefsHandler::timeZoneListAsJson(const std::string& countryCode, cons
 		timeZonesListObj.put("timeZone", timeZoneArray);
 		if (countryCode.empty()) {
 			timeZonesListObj.put("syszones", sysZones);
-			timeZonesListObj.put("mmcInfo", mmcInfoObj);
+			timeZonesListObj.put("mccInfo", mccInfoObj);
 		}
 
 		if (!timeZonesListObj.isNull()) {
